tasks/ubooquity.yml

```diff
@@ -0,0 +1,33 @@
+---
+- name: Create Ubooquity Directories
+  file:
+    path: "{{ item }}"
+    state: directory
+  with_items:
+    - "{{ ubooquity_data_directory }}"
+
+- name: Ubooquity Docker Container
+  docker_container:
+    name: ubooquity
+    image: linuxserver/ubooquity:latest
+    pull: true
+    volumes:
+      - "{{ ubooquity_data_directory }}/config:/config:rw"
+      - "{{ ubooquity_data_directory }}/files:/files:rw"
+      - "{{ comics_root }}:/comics"
+      - "{{ books_root }}:/books"
+    env:
+      TZ: "{{ ansible_nas_timezone }}"
+      PUID: "{{ ubooquity_user_id|quote }}"
+      PGID: "{{ ubooquity_group_id|quote }}"
+      MAXMEM: "1024"
+    ports:
+      - "2202:2202"
+      - "2203:2203"
+    restart_policy: unless-stopped
+    #memory: 1g
```

Comment:
Is there a reason why the memory limit is commented out?

Reply:
If you take a look at the environment variables you will see that the application is limited via this variable. I have been running it exactly like this with no issues for nearly a year now. Woud you like for me to just remove the text on line 28?